forms-flow-web/src/apiManager/services/bpmServices.js

```diff
@@ -73,3 +77,18 @@ export const getPaginatedForms = (forms, limit, page, sort) => {
   forms.sort(dynamicSort(sort));
   return forms.slice((page - 1) * limit, ((page - 1) * limit) + limit);
 };
+
+export const deployBpmnDiagram = (data, token, isBearer = true) => {
+
+  const url = BPM_BASE_URL + "/deployment/create";
```

Comment:
Move base url Data to API Url configs and import

Reply:
https://github.com/AOT-Technologies/forms-flow-ai/blob/master/forms-flow-web/src/apiManager/endpoints/index.js

Reply:
Done